<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rgb_na_cmyk</class>
 <widget class="QWidget" name="rgb_na_cmyk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="horizontalSlider_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>szarobłękitny lub sinoniebieski.</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_m">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>kolor uzyskany w wyniku połączenia czerwieni i niebieskiego.</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_y">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>kolor bardzo podobny do żółtego.</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="anuluj">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Wyzerowanie suwaków</string>
   </property>
   <property name="text">
    <string>Zerowanie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_c">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_m">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="cyjan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>szarobłękitny lub sinoniebieski.</string>
   </property>
   <property name="text">
    <string>Cyjan</string>
   </property>
  </widget>
  <widget class="QLabel" name="magenta">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>kolor uzyskany w wyniku połączenia czerwieni i niebieskiego.</string>
   </property>
   <property name="text">
    <string>Magenta</string>
   </property>
  </widget>
  <widget class="QLabel" name="yellow">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>kolor bardzo podobny do żółtego.</string>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_k">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>kolor czarny.</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="black">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>kolor czarny.</string>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
